src/resolve_robotics_uri_py/resolve_robotics_uri_py.py

```diff
@@ -76,19 +76,31 @@ def pathlist_list_to_string(path_list: Iterable[str | pathlib.Path]) -> str:
 # ===================
 
 
-def resolve_robotics_uri(uri: str) -> pathlib.Path:
+def resolve_robotics_uri(
+    uri: str, package_dirs: list[str] | None = None
+) -> pathlib.Path:
     """
     Resolve a robotics URI to an absolute filename.
 
     Args:
         uri: The URI to resolve.
+        package_dirs: A list of additional paths to look for the file.
 
     Returns:
         The absolute filename corresponding to the URI.
 
     Raises:
         FileNotFoundError: If no file corresponding to the URI is found.
+
+    Note:
+        In case `package_dirs` is not provided, the function will look for the file in the
+        default search paths specified by the environment variables in `SupportedEnvVars`.
+
+        If a given model searches for the meshes in `package://ModelName/meshes/mesh.stl`, and the
+        actual mesh is in `/usr/local/share/ModelName/meshes/mesh.stl`, the `package_dirs` should
+        contain `/usr/local/share`.

```

Comment:
The sentence is not clear to me, as the default search paths are searched even when `package_dirs` is provided. My proposal:

```suggestion
        By default the function will look for the file in the
        default search paths specified by the environment variables in `SupportedEnvVars`.
 
        If the `package_dirs` argument is provided, the model is also searched in the folders
        specified in `package_dirs` . In particular if a file is specified by the uri
        `package://ModelName/meshes/mesh.stl`, and the actual file is in 
        `/usr/local/share/ModelName/meshes/mesh.stl`, the `package_dirs` should contain `/usr/local/share`.
```

Reply:
It looks good, thanks!